documentation/src/main/resources/openapi/ditto-api-2.yml

```diff
@@ -10430,6 +10455,43 @@ components:
         - security
         - securityDefinitions
       additionalProperties: true
+    PermissionCheckRequest:
+      type: object
+      description: "Request to check permissions for various entities and resources."
+      additionalProperties:
+        $ref: '#/components/schemas/PermissionEntityCheck'
+      example:
+        lamp_reader:
+          resource: "thing:/features/lamp/properties/on"
+          entityId: "org.eclipse.ditto:some-thing-1"
+          hasPermissions: [ "READ" ]
+        lamp_toggler:
+          resource: "message:/features/lamp/inbox/toggle"
+          entityId: "org.eclipse.ditto:some-thing-1"
+          hasPermissions: [ "WRITE" ]
+    PermissionEntityCheck:
+      type: object
+      description: "Details for a specific permission check request."
+      properties:
+        resource:
+          type: string
+          description: "Resource path the permission check applies to."
```

Comment:
Maybe we could be more specific here about the format of that `resource` .. Or even provide a regex `pattern`?